src/System.IO.Ports/ref/Configurations.props

```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
-      netfx;
-      net461;
+      netfx-Windows_NT;
```

Comment:
Reference assemblies cannot have runtimes.

Reply:
Maybe we should add an error for this case into some common targets file.

Reply:
done.

Reply:
Central error so others don't make the same mistake?

Reply:
https://github.com/dotnet/corefx/issues/36568